package/yast2-bootloader.changes

```diff
@@ -1,50 +1,98 @@
 -------------------------------------------------------------------
+Wed Aug  2 16:00:44 CEST 2017 - snwint@suse.de
+
+- not everything is a disk, use libstorage's partitionable instead
+- 3.4.1
+
+-------------------------------------------------------------------
+Mon Jul 24 16:05:52 UTC 2017 - ancor@suse.com
+
+- storage-ng: adapted calls to temporary y2storage_* methods, since
+  they were finally removed from Y2Storage::StorageManager.
+
+-------------------------------------------------------------------
+Mon Jul 10 15:41:22 UTC 2017 - jlopez@suse.com
+
+- storage-ng: adapt detection of PReP partition
+
+-------------------------------------------------------------------
+Wed Jul  5 12:27:37 CEST 2017 - shundhammer@suse.de
+
+- Merged master to storage-ng branch
+- 3.4.0
+
+-------------------------------------------------------------------
+Tue Jul  4 15:27:27 UTC 2017 - igonzalezsosa@suse.com
+
+- Adapt the module to use the new storage-ng during system cloning
+  (bsc#1047245)
+- 3.3.9
+-------------------------------------------------------------------
 Fri Jun 23 09:14:05 CEST 2017 - schubi@suse.de
 
 - AutoYaST configuration: Set StorageDevices flag disks_valid
   to true. So libstorage can scan valid disks and "/" partition
   can be found. (bnc#1046738, bnc#1043132)
-- 3.3.0
 
 -------------------------------------------------------------------
 Wed Jun 14 14:54:38 UTC 2017 - jreidinger@suse.com
 
 - Use udev device for prep partition if it is available
   (bsc#1041692)
-- 3.2.22
```

Comment:
As discussed in irc, loosing this kind of information or adding entries down there in the changelog is a no-go for me from the package history point of view.